lightning/src/offers/offer.rs

```diff
@@ -1199,6 +1204,9 @@ mod tests {
 			.unwrap();
 		#[cfg(feature = "std")]
 		assert!(!offer.is_expired());
+		let now = std::time::SystemTime::UNIX_EPOCH
+			.elapsed().unwrap();
```

Comment:
This won't work when the test is run in `no-std`. Instead, assign `now` by subtracting some time from `future_expiry`.